docs/index.html

```diff
@@ -466,6 +491,13 @@ <h2>#TwitterThrowback</h2>
         }, 1000);
     }
 
+    //Collapsible Effect
+    function minimizer(event){
```

Comment:
It's a function that does something. Please name it with a verb (like `minimize`). But I'd suggest something more accurate and descriptive, like `toggleTweetsView` or even `showOrHideTweetsInYear`. Long, but descriptive.

```suggestion
    function showOrHideTweetsInYear(event){
```

Reply:
Ok. Sure.